### PR TITLE
LPS-102898 SF

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_item_selector_preview.scss [hunk 1]
```diff
@@ -62,6 +64,16 @@
...
 			color: #fff;
+			height: calc(100% - 7rem);
+			text-align: left;
+
+			.nav-link.active {
+				color: #fff;
+			}
+
+			.sidebar-dd {
+				color: #a7a9bc;
```
Comment (boton) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_item_selector_preview.scss]: Maybe use atlas vars is a better solution than hardcoded colors

**Comment by brianchandotcom** [comment]
@ambrinchaudhary please resend  based on feedback from @boton

**Comment by ambrinchaudhary** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/80153

**Pull request closed without merging**